# see https://github.com/RIVeR-Lab/apriltags_ros/blob/indigo-devel/apriltags_ros/msg/AprilTagDetection.msg
# TODO: make general with string parsing
string algorithm
int32 id
# float64 size
geometry_msgs/PoseStamped pose